libraries/botbuilder/src/streaming/streamingHttpClient.ts

```diff
@@ -34,8 +34,12 @@ export class StreamingHttpClient implements HttpClient {
      */
     public async sendRequest(httpRequest: WebResource): Promise<HttpOperationResponse> {
         if (!httpRequest) {
-            throw new Error('SendRequest invalid parameter: httpRequest should be provided');
+            throw new Error('StreamingHttpClient.sendRequest(): missing "httpRequest" parameter');
         }
+        if (!this.server.isConnected) {
+            throw new Error('StreamingHttpClient.sendRequest(): Streaming connection is disconnected');

```

Comment:
```suggestion
            throw new Error('StreamingHttpClient.sendRequest(): Streaming connection is disconnected, and the request could not be sent.');

```

Would be good to explain why it's a problem in the error message